packages/@aws-cdk/aws-ec2/lib/vpc.ts

```diff
@@ -562,14 +566,15 @@ export interface VpcAttributes {
 
 export interface SubnetAttributes {
   /**
-   * The Availability Zone the subnet is located in
+   * The subnetId for this particular subnet
```

Comment:
Changing the order of these properties just adds a diff for no good reason.

Reply:
reverted these changes